portfolio/src/main/webapp/script.js

```diff
@@ -65,13 +65,16 @@ async function fetchContentFromServlet() {
     const response = await fetch('/data');
     const arr = await response.json();
 
-    const stringList = document.getElementById('week2-fetch');
-    stringList.innerHTML = '';
+    const commentsList = document.getElementById('week2-fetch');
 
-    arr.length === 0 ? stringList.innerHTML = '<h3>No Comments Yet</h3>' : stringList.innerHTML = '<h3>Comments:</h3>';
+    arr.length === 0 ? commentsList.innerHTML = '<h3>No Comments Yet</h3>' : commentsList.innerHTML = '<h3>Comments:</h3>';
 
+    const unorderedList = document.createElement('ul');
```

Comment:
unorderedList is a too abstract. How about calling it commentListElement?

It's useful to highlight that it is used to store commentList, and it is an HTML element. The existing commentsList can be called commentListDiv to better differentiate them.

Reply:
got it!